lte/gateway/c/session_manager/test/ProtobufCreators.cpp

```diff
@@ -15,6 +15,42 @@
 
 namespace magma {
 
+void build_common_context(
```

Comment:
You don't need to change this because I don't know if what I am going to say is right: couldn't we instead of passing a reference to `common_context` just return a pointer to the object? Or if you don't want to return a pointer because cpp doesn't recommend them, why not just return `common_context` itself (not really big object so return will copy it but will not cause big issue). 

I am just saying because I see this pattern of creating the object outside the function, passing as an argument, and filling it inside the function all over the place. If we could just return the object itself that would make its readability easier.... 

But again, maybe I am saying something totally against cpp recommendations

Reply:
Yeah, I kind of just followed the pattern of all the other functions in this file. :) Let me look at the common pattern for C++ for these kinds of things. :)

Reply:
Thanks, that will help me too!!
I did some research some time and found out return by value may not cause a copy due to `copy elision`. So maybe the best is just return it by value and let the compiler to optimize

https://www.quora.com/Is-there-any-gorgeous-way-for-C++-to-return-an-object-without-copying-it-in-memory